src/pages/PoolFinder/index.tsx

```diff
@@ -112,6 +112,8 @@ export default function PoolFinder() {
     mixpanelHandler(MIXPANEL_TYPE.IMPORT_POOL_INITIATED)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
+
+  if (!isEVM) return <Redirect to="/" />
```

Comment:
should create a Layout to handle this instead of manually check in every page

Reply:
How would you suggest